site/docs/v1/tech/release-notes.adoc

```diff
@@ -49,21 +49,68 @@ include::docs/v1/tech/__database-migration-warning.adoc[]
 
 === New
 * JWT Vending machine
-* IP Access Control. Requires Threat Detection feature.
+** This allows a JWT to be created for a not-yet-existing user with a payload defined by the API caller.
+** Resolves https://github.com/FusionAuth/fusionauth-issues/issues/525[GitHub Issue #525]
+* FusionAuth wasn't awesome enough, so we added a robust Threat Detection feature for enterprise customers. This feature includes: 
+* IP Access Control for API keys
+** This allows domains or API addresses to be safelisted to use an API key.
+** Resolves https://github.com/FusionAuth/fusionauth-issues/issues/933[GitHub Issue #933]
+*** Rate Limiting
+*** Captcha
+*** IP Access Control Lists/blocking
+*** Blocked domains
 * Rate limiting per user for the following requests:
 ** Failed login (only used if Failed Login configuration is not in use)
 ** Forgot password
 ** Send email verification
 ** Send passwordless
 ** Send registration verification
 ** Send two-factor
-* Captcha. Requires Threat Detection feature.
-** Support for Google ReCaptcha v2, Google ReCaptcha v3, HCaptcha and HCaptcha Enterprise
-* IP location. Requires Threat Detection feature.
-* Impossible travel calculation during login. Requires Threat Detection feature.
-* More emails
-* More webhook events
-** Kickstart Success `kickstart.success`
+* Captcha - uses additional Threat Detection feature.
```

Comment:
uses additional Threat Detection feature. -> Sounds strange.  What is the meaning?